api/session/sessionRouter.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const authRequired = require('../middleware/authRequired');
+const Session = require('./sessionModel');
+const router = express.Router();
+// eslint-disable-next-line
+router.get('/', authRequired, async function (req, res, next) {
```

Comment:
We could implement the roleAuth middleware, and give access to instructors/admin.